types/hapi__subtext/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for @hapi/subtext 7.0.3
+// Project: https://github.com/hapijs/subtext#readme
+// Definitions by: Sebastian Malton <https://github.com/nokel81>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { IncomingMessage } from 'http';
+import stream from 'stream';
+import { BrotliOptions, ZlibOptions } from 'zlib';
+
+export interface MultipartOptions {
+  output: "data" | "stream" | "file";
+}
+
+
+export interface DecompressionOptions {
+  gzip?: ZlibOptions;
+  deflate?: ZlibOptions;
+  br?: BrotliOptions;
+  compress?: BrotliOptions;
+}
+
+export interface ContentDecoders {
+  gzip?: (options: ZlibOptions | null) => stream.Transform;
+  deflate?: (options: ZlibOptions | null) => stream.Transform;
+  br?: (options: BrotliOptions | null) => stream.Transform;
+  compress?: (options: BrotliOptions | null) => stream.Transform;
+}
+
+export interface Options {
+  parse: boolean;
+  output: "data" | "stream" | "file";
+  maxBytes?: number;
+  override?: string;
+  defaultContentType?: string;
+  allow?: string[];
+  timeout?: number;
+  querystring?: (str: string) => Partial<Record<string, string | string[]>>;
```

Comment:
For instance the documentation states that `qs` is an option but nowhere in the library code is that field used and the `querystring` field is actually used. This field is an way to override the parsing of querystrings.